source/jormungandr/readme.md

```diff
@@ -0,0 +1,114 @@
+
+# Jormungandr
+Web service around the c++ core(s). This is the navitia api front end, based on flask.
+
+:snake: **python module**
+
+## Norse mythology
+In the Norse mythology, Jörmungandr is a sea serpent that grew so large that it can surround the earth, hence his name: Midgard Serpent. He will be killed by Thor during Ragnarok.
+
+# Setup
+## VirtualEnv
+
+```sh
+cd navitia/source/jormungandr
+mkvirtualenv jormungandr
+pip install -r requirements.txt -r requirements_dev.txt -U
+```
+
+# Configure
+
+Jormungandr uses 2 different levels of config:
+* `jormungandr.json` : that describes Kraken's instances and coverages
+* Jormungandr's core config via :
+	* Environment variables [the prefered option]  
+	* A custom setting file `custom_settings.py` [for advanced usage] - that overwrites default settings
+
+## jormungandr.json
+
+Create a file `jormungandr.json` in a custom directory like:
+
+```json
+{
+    "key": "some_region",
+    "socket": "ipc:///tmp/default_kraken"
+}
+```
+
+You can copy the one from [documentation/examples/config/Jormungandr.json](https://github.com/CanalTP/navitia/blob/dev/documentation/examples/config/Jormungandr.json)
+
+Make sure `key` and `socket` respectively match `instance_name` and `zmp_socket` from your Kraken config file ([kraken.ini](https://github.com/CanalTP/navitia/blob/dev/documentation/examples/config/kraken.ini)).
+
+## Environement Variables
+
+Environment variables can be set when the service is started. Referer to [default_settings.py](https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/default_settings.py)
+
+example for a devlopment environment: 
+
+```sh
+PYTHONPATH=..:../../navitiacommon/ JORMUNGANDR_INSTANCES_DIR=~/jormung_conf/ JORMUNGANDR_START_MONITORING_THREAD=False  JORMUNGANDR_DISABLE_DATABASE=True JORMUNGANDR_IS_PUBLIC=True python manage.py runserver
```

Comment:
Maybe talk about json in instance env var?

Reply:
now added in a685f32